packages/ubuntu_desktop_installer/lib/pages/who_are_you/who_are_you_model.dart

```diff
@@ -106,9 +106,9 @@ class WhoAreYouModel extends ChangeNotifier {
   /// Loads the identity data from the server, and resolves the system hostname.
   Future<void> loadIdentity() async {
     final identity = await _client.identity();
-    _realName.value = identity.realname?.orIfEmpty(_realName.value);
-    _hostname.value = identity.hostname?.orIfEmpty(_hostname.value);
-    _username.value = identity.username?.orIfEmpty(_username.value);
+    _realName.value ??= identity.realname?.orIfEmpty(null);
+    _hostname.value ??= identity.hostname?.orIfEmpty(null);
+    _username.value ??= identity.username?.orIfEmpty(null);
```

Comment:
The `??=` indeed makes more sense. Now we have the existing value, if set, as the top precedence, the result from the client call or null at last. Correct?

Reply:
Yep! And empty values from the client are kept as null to distinguish between initially empty vs. manually erased empty states.